saleor/order/tasks.py

```diff
@@ -38,7 +38,7 @@ def recalculate_orders_task(order_ids: list[int]):
 
 @app.task
 def send_order_updated(order_ids):
-    manager = get_plugins_manager()
+    manager = get_plugins_manager(allow_replica=False)
```

Comment:
As above. It is okay to have `allow_replica=True` in any celery task which initialize the manager.